### PR TITLE
Updating fcpgenkey, fcpinvertkey man pages to 0.2.5

### PR DESCRIPTION
**Pull request by Soliloque**
Then a few cosmetic changes and modifying other man pages to keep those uniform.

Removed html version of man pages as those are 
1) redondant
2) obsolete
3) automaticaly generated from the other man pages version.

Valided original author's email adress.

### CODE REVIEW

**Comment by Thynix** [comment]
Ooh, getting rid of those HTML versions will be nice. Some of these commit messages do not meet the [commit message standards](https://wiki.freenetproject.org/Coding_standards#Commit_Messages). Could you please update them?

**Comment by Soliloque** [comment]
Done.

File: AUTHORS [hunk 1]
```diff
@@ -2,5 +2,5 @@ The PyFCP applications and library were written
 by David McNab (aum) in May-June, 2006.
 
 Contact the author at:
-  david@freenet.org.nz
+  david@rebirthing.co.nz
```
Comment (Thynix) [inline AUTHORS]: Where did this email come from?

Comment (Soliloque) [inline AUTHORS]: From the man pages. Then I checked http://rebirthing.co.nz in search of this email adress. Did not find it but find a picture of David McNab on the first page. I write him to be certain. No reply yet but I did not receive any error. So this email is most probably working.

Comment (Thynix) [inline AUTHORS]: I wonder if David still wants to be listed as maintainer for this codebase. I suspect not.

**Comment by Thynix** [comment]
The way the author sections are written makes it sound like this was written in one go in 2006, and hasn't been changed since, which isn't the case.

**Comment by Soliloque** [comment]
It was **spelt** wrong. ;)

**Comment by Thynix** [comment]
I'm sorry, these still don't meet the commit message standards. Are they unclear? I'd like to clarify them - any suggestions?

For example, in `pyFreenet man pages: Change SYNTAX->SYNOPSIS` the context doesn't need to include "pyFreenet" because it is already in the pyFreenet repo. The standards suggest a context of the "area / module" if helpful, which could mean a class but in this case means the start of the path. Words are also preferable to symbols, so I'd recommend changing this one to `manpages: rename SYNTAX section to SYNOPSIS`.

Commit summaries (meaning the first line) should not end with a full stop.

The rest are not imperative - meaning commands to the codebase:
* updating -> update
* new -> add
* Removed -> Remove
* AUTHOR sections expanded -> expand AUTHOR sections

Thanks!

**Comment by Soliloque** [comment]
English isn't my native language. The PyFCP applications and library were originaly written by David McNab (aum) in May-June, 2006 and are maintained by the Freenet Project team.

**Comment by Thynix** [comment]
Alright - sorry about that. Would you like me to merge once the content looks good and fix the commit messages where necessary?

That sounds good for the authors section. originaly -> originally. (I assume you plan to do the  pyFCP -> pyFreenet later?)

**Comment by Soliloque** [comment]
Answering to a lot of things at one.

* I'm fixing everything right now. I don't want to give you too much job.

* As I wrote him, I suppose McNab will tell me if he doesn't want to be a maintainer anymore. I'll remove him if needed. 

* I don't want to replace pyFCP by pyFreenet only on a few files and further add to the imprecision.

**Comment by Soliloque** [comment]
I think it begins to look good. Please notice the last commit and the ones about AUTHORS and email.

File: manpages/fcpgenkey.1 [hunk 2]
```diff
@@ -11,9 +11,28 @@ fcpgenkey \- generate a single Freenet SSK@/USK@ keypair
...
-The keypair generated is written to standard output stream, with
-the public key, then a newline, then the private key.
+The keypair generated is written to standard output stream as SSK@
+keys, with the public key, then a newline, then the private key.
 
+.SH "Converting an SSK@ key to an USK@ one"
+.LP
+USK@ are really just a user-friendly wrapper around SSK@, which hide
+the process of searching for more recent versions of a site. Any
+address written using an SSK@ key can also be written using an USK@
```
Comment (Thynix) [inline manpages/fcpgenkey.1]: This way isn't strictly correct, but the other way is. Any USK can be written as an SSK, but not every SSK can be written as a USK.

File: manpages/fcpgenkey.1 [hunk 3]
```diff
@@ -11,9 +11,28 @@ fcpgenkey \- generate a single Freenet SSK@/USK@ keypair
...
+http://127.0.0.1:8888/SSK@<key>/<name_of_site>-<version#>/
+.TP
+.B
+Syntax of address using USK@ key:
+http://127.0.0.1:8888/USK@<key>/<name_of_site>/<version#>/
+.TP
+.B
+These two addresses are completely equivalent and can be inserted as either one:
+http://localhost:8888/USK@z6lnO...ACAAE/freenetdocwiki/17/
+http://localhost:8888/SSK@z6lnO...ACAAE/freenetdocwiki-17/
```
Comment (Thynix) [inline manpages/fcpgenkey.1]: Please remove the `http://localhost:8888/`s - it is not part of the key.

File: manpages/fcpgenkey.1 [hunk 4]
```diff
@@ -11,9 +11,28 @@ fcpgenkey \- generate a single Freenet SSK@/USK@ keypair
 fcpgenkey is a simple command\-line FCP client program for generating
 new Freenet SSK@ or USK@ public/private keypairs.
 
-The keypair generated is written to standard output stream, with
-the public key, then a newline, then the private key.
+The keypair generated is written to standard output stream as SSK@
+keys, with the public key, then a newline, then the private key.
 
+.SH "Converting an SSK@ key to an USK@ one"
```
Comment (Thynix) [inline manpages/fcpgenkey.1]: As below the title should be the other way around, I think.

File: manpages/fcpgenkey.1 (hunk 2 above)
Comment (Soliloque) [inline manpages/fcpgenkey.1]: I agree. I'll reverse it.

File: manpages/fcpgenkey.1 (hunk 3 above)
Comment (Soliloque) [inline manpages/fcpgenkey.1]: I agree but I'll exchange "address" for "key", too.

File: manpages/fcpgenkey.1 (hunk 4 above)
Comment (Soliloque) [inline manpages/fcpgenkey.1]: I disagree on this one. From the POV of a user of fcpgenkey, fcpgenkey provides him with a SSK@ key but he really want a USK@ key so he needs to understand they are equivalent for his purpose.

Comment (Thynix) [inline manpages/fcpgenkey.1]: Okay, then how about "Equivalence between SSKs and USKs"?

File: manpages/fcpgenkey.1 [hunk 5]
```diff
@@ -11,9 +11,28 @@ fcpgenkey \- generate a single Freenet SSK@/USK@ keypair
...
+.LP
+USK@ are really just a user-friendly wrapper around SSK@, which hide
+the process of searching for more recent versions of a site. Any
+address written using an USK@ key can also be written using an SSK@
+key.
+.TP
+.B
+Syntax of SSK@ key:
+SSK@<key>/<name_of_site>-<version#>/
```
Comment (Thynix) [inline manpages/fcpgenkey.1]: How about the USK goes first and then the SSK is "Syntax of equivalent SSK@ key:"? SSKs don't have to be formatted that way.

Comment (Soliloque) [inline manpages/fcpgenkey.1]: done

**Comment by Thynix** [comment]
Looks good, thanks!

**Comment by Soliloque** [comment]
Thank you for your patience.

**Comment by Thynix** [comment]
Up to you - I should get an email, but if it looks like I've forgotten feel free to poke me about it.

**Pull request merged**